config/locales/en.yml

```diff
@@ -1187,69 +1187,183 @@ en:
       stimulus_payment:
         faq:
           eip:
-            additional_eip:
-              a: In most cases, the additional EIP would be provided in 2021.
-              q: What if I missed the deadline to provide additional information or received a smaller EIP than expected?
-            deadline:
-              a: For those concerned about visiting a tax professional or local community organization in person to get help with a tax return, these economic impact payments will be available throughout 2020 and the 2021 tax filing season.
-              q: I still need to file a tax return. How long are the economic impact payments available?
             header: Economic Impact Payment (stimulus payment) FAQs
-            one_time:
-              a: Only one stimulus package has been announced by the U.S. government. We will update this page with any changes.
-              q: Is this a one-time payment?
-            other_benefits:
-              a: No, the EIP checks are treated as a tax credit. This means your payment will not impact the benefits you receive now or in the future.
-              q: Will the EIP check affect my other government benefits?
-            owe_taxes:
-              a: Yes, but with one exception. The Payment will be offset by past-due child support. The Bureau of the Fiscal Service will send you a notice if this occurs.
-              q: If I owe taxes, or have a payment agreement with the IRS, or owe other federal or state debts, will I still get the EIP?
-            scams:
-              a: Don’t click on links whose origins you aren’t 100% sure of. Be wary of scams. If it looks too good to be true, it probably is. And don’t send money to someone you don’t know.
-              q: What if someone offers a quicker payment?
-            when:
-              a: Economic Impact Payments will arrive in bank accounts through direct deposit starting on April 9th, 2020. If you don't have a bank account it could take up to five months to receive your check by mail.
-              q: When will I receive my check?
+            items:
+              - title: When will I receive my check?
+                content_html: >-
+                  The second Economic Impact Payments were sent beginning in December 2020.
+                  If you have not received your payment you can check the
+                  status on the IRS <a
+                    href="https://www.irs.gov/coronavirus/get-my-payment"
+                    rel="nofollow"
```

Comment:
I recommend `rel="nofollow noopener`" to avoid a weird HTML gotcha; see https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener